osu.Game/Users/User.cs

```diff
@@ -249,6 +282,14 @@ public class UserHistoryCount
             Id = 0
         };
 
+        public bool Equals(User other)
```

Comment:
Why was this moved from the end of the file?

Reply:
was re-ordering the class a bit, made it to being "properties then user logic then nested classes and enums".

Reply:
We generally put all the system functions right at the bottom (dispose/equality/etc).